### DISCUSSION TITLE
Ability for Form Placeholder to poll

### DISCUSSION THREAD

**Discussion by SimonMacIntyre** [filamentphp/filament#13459] [Ideas]
We make use of Placeholder in forms, to show data related to the entity. For example let's say we have an EditUser page and form. 

We are using Placeholder as an example while viewing this page, to show some interesting live updating status about some thing based on this user. (Note: This is not suitable to be in a table/relation manager, we make heavy use of that too, but this specifically is a component we want to show right up in the form in a read-only fashion, but have it live-update).

Any chance of this being a feature we can add some day? Ideally it would be as simple as... 
```php
Placeholder::make('status')->poll(5)->...
```

**Comment by SimonMacIntyre** [comment]
@danharrin  If you think this is feasible at all, would be happy to pitch in a bit to a `polar` fund to incentivize if you can make it into an `issue` instead of a discussion. Let me know though if you think this idea has any merit from your perspective.

**Reply by danharrin** [reply]
Sure, I'll convert it into an issue, but it will be a v4 feature, because we'll be able to implement it in a more performant way (only the placeholder / infolist entry will render, not the entire form/infolist) which isnt possible in v3

**Reply by SimonMacIntyre** [reply]
Thanks Dan!

**Comment by indigoram89** [comment]
Hi guys, thank you for your work! Can I poll entire infolist in v3?

**Reply by leandrocfe** [reply]
You could customize the view https://filamentphp.com/docs/3.x/panels/resources/viewing-records#custom-view
and use `wire:poll`

There are render hooks that you can use to achieve it, if you don't want to customize the view

**Reply by indigoram89** [reply]
Wow! I understood, thank you and good luck!

**Reply by indigoram89** [reply]
Another solution is to add ViewEntry to infolist with div[wire:poll] and it works only for infolist layout.

![CleanShot 2024-08-08 at 17 02 56@2x](https://github.com/user-attachments/assets/57899395-8232-475c-91f2-b5cbfb413ff8)
